benchmark/benchmark_json/src/BM_Cereal_vs_Simdjson_Test.cpp

```diff
@@ -0,0 +1,525 @@
+/****************************************************************************
+ *
+ *                           Klepsydra Core Modules
+ *              Copyright (C) 2019-2020  Klepsydra Technologies GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *****************************************************************************/
+#define CEREAL_THREAD_SAFE 1
+
+#include <benchmark/benchmark.h>
+#include <iostream>
+#include <simdjson.h>
+#include <vector>
+#include <limits>
+#include <random>
+#include <fstream>  // ofstream
+
+#include <cereal/archives/json.hpp>
+#include <cereal/cereal.hpp>
+#include <cereal/types/vector.hpp>
+
+
+// ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- 
+// ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- 
+bool save_to_myfile(const std::string & str, const std::string & fileName)
```

Comment:
Move all static methods to a new class with static methods in a class.
Call it `SerializationBenchmarkUtils`
Move it to its own file